gplugins/lumerical/Lumerical.py

```diff
@@ -1,21 +1,26 @@
-from enum import Enum
-import time
 import shutil
-import yaml
+import time
+from enum import Enum
 
-import lumapi 
-import numpy as np
 import gdsfactory as gf
+import lumapi
+import numpy as np
+import yaml
+from gdsfactory.config import PathType, logger
+from gdsfactory.generic_tech.simulation_settings import (  # TODO wildcard imports are bad
+    SIMULATION_SETTINGS_LUMERICAL_FDTD,
+    SimulationSettingsLumerical,
+    SimulationSettingsLumericalFdtd,
+)
+from gdsfactory.pdk import get_layer_stack
+from gdsfactory.typings import Component, LayerStack, MaterialSpec
 
-from gdsfactory.pdk import ComponentSpec, get_layer_stack
 from gplugins.common.utils.get_sparameters_path import (
     get_sparameters_path_lumerical as get_sparameters_path,
 )
-from gdsfactory.typings import Component, LayerStack, MaterialSpec
-from gdsfactory.config import PathType, __version__, logger
-from gdsfactory.generic_tech.simulation_settings import SimulationSettingsLumerical, SimulationSettingsLumericalFdtd, SIMULATION_SETTINGS_LUMERICAL_FDTD # TODO wildcard imports are bad
-
-from gplugins.lumerical.write_charge_distribution_lumerical import add_charge_materials, add_doping, write_charge_distribution_lumerical
+from gplugins.lumerical.write_charge_distribution_lumerical import (
+    add_charge_materials,
+)
 from gplugins.lumerical.write_sparameters_lumerical import set_material
 
 DEBUG_LUMERICAL = 1
```

Comment:
comment? "This enables/disables showing the lumerical GUI when interacting with sessions"